packages/cml/src/replaceCmlPlaceholders/replaceCmlPlaceholders.ts

```diff
@@ -77,7 +77,7 @@ const replacePropValues = (mdx: string, values: Record<string, string> = {}): st
       case 'boolean':
       case 'number':
         searchValue = match;
-        newValue = `{${propValue}}`;
+        newValue = `{${propValue ?? defaultValue}}`;
```

Comment:
question, the case is wrapping number too, are we sure that using nullish coalescing is ok? because if the value is 0 it will return defaultValue instead. Iono if that's the desired behavior. Otherwise `||` will do it.

Reply:
As far as I know nullish coalescing operator (??) only checks if the first operand is null or undefined, but I'll add a test to cover that scenario too.

Reply:
yeah that's true. ?? works ok in this case. I reversed the behaviors between operators 😅